client/components/auth-form.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+export default class AuthForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      email: '',
+      password: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch('/api/users/sign-up', req)
+      .then(res => res.json())
+      .then(result => {
+      });
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  }
+
+  render() {
+    return (
+      <>
+        <form onSubmit={this.handleSubmit}>
+        <div className="auth-modal-content">
+            <i onClick={this.props.onAuthClick} className="fa-solid fa-xmark"></i>
+            <h1 className='no-margin '>BOOK<span className='header-lover'>LOVER</span></h1>
+            <h3 className='sing-up-header'>Sign up</h3>
+            <p className='no-margin sing-up-text'>Sign up to see the latest bestsellers and search your favorite authors and books.</p>
+              <label htmlFor="username"><p className='auth-margin'>Your name</p></label>
+              <input onChange={this.handleChange} type="text" placeholder="Name" name="username" id="username" required />
+
+              <label htmlFor="email"><p className='auth-margin'>Email address</p></label>
+              <input onChange={this.handleChange} type="text" placeholder="exapmle@email.com" name="email" id="email" required />
+
+              <label htmlFor="password"><p className='auth-margin'>Create password</p></label>
+              <input onChange={this.handleChange} type="password" placeholder="At least 8 characters" name="password" id="password" required />
+
+              <div className="button-holder">
+                <button type="submit" className="auth-button">Sign Up</button>
+              <p className='sing-up-text no-margin'>Already have an account?<a className='log-in-a' href="#logn-in">Logn in</a></p>
+              </div>
+          </div>
+        </form>
+      </>
+    );
```

Comment:
You can remove the react fragment here.

Reply:
<img width="190" alt="Screen Shot 2022-09-16 at 9 58 40 AM" src="https://user-images.githubusercontent.com/106217355/190690851-cd7767aa-eef4-4587-8b3c-0320847cd149.png">